recipes-bsp/uefi/edk2-hikey960/grub.cfg.in

```diff
@@ -0,0 +1,12 @@
+set default="0"
+set timeout=1
+
+menuentry 'CE Reference Platform (HiKey960 @DISTRO)' {
+    linux /boot/@KERNEL_IMAGETYPE console=tty0 @CMDLINE
+    devicetree /boot/hi3660-hikey960.dtb
+}
+
+menuentry 'Fastboot' {
+    search.fs_label boot boot_part
+    chainloader ($boot_part)/EFI/BOOT/fastboot.efi
+}
```

Comment:
Do we need to provide "Fastboot" in grub? Maybe we can use "Fastboot" in boot manager stage since we support hotkey to invoke "Fastboot".